Files/ViewModels/SettingsViewModel.cs

```diff
@@ -81,6 +81,7 @@ private void TrackAnalytics()
             Analytics.TrackEvent($"{nameof(ShowLibrarySection)} {ShowLibrarySection}");
             Analytics.TrackEvent($"{nameof(ShowBundlesWidget)} {ShowBundlesWidget}");
             Analytics.TrackEvent($"{nameof(ListAndSortDirectoriesAlongsideFiles)} {ListAndSortDirectoriesAlongsideFiles}");
+            Analytics.TrackEvent($"{nameof(AreFileTagsEnabled)} {AreFileTagsEnabled}");
```

Comment:
Should this be changed to `AreTagsEnabled` since tags work for folders as well?

Reply:
I'd leave the variables name as they are, if any we could change the user-facing string resources to something different than "file tags", but just "tags" doesn't sound clear 🤔

Reply:
I'm not sure file tags is much clearer 🤔. Perhaps we should add information to the docs or a teaching tip to the app.